experimental/packages/opentelemetry-sdk-metrics-base/src/MeterProvider.ts

```diff
@@ -69,9 +136,37 @@ export class MeterProvider implements metrics.MeterProvider {
     this._sharedState.metricCollectors.push(collector);
   }
 
-  addView(view: View, instrumentSelector: InstrumentSelector, meterSelector: MeterSelector) {
-    // https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#view
-    this._sharedState.viewRegistry.addView(view, instrumentSelector, meterSelector);
+  addView(options: ViewOptions) {
```

Comment:
What about splitting the view options into two parts: one for the (not optional) view configurations, and one for the (optional) selectors? The view configurations are not optional since we are adding the view. The selectors can be optional as we can set the view as the default view.

In this way, we don't nest view configuration keys like `description`, `aggregation` etc, in `ViewOptions.stream`.

Reply:
Thanks for the feedback, I like that idea. Changed in [1beea85](https://github.com/open-telemetry/opentelemetry-js/pull/2820/commits/1beea85f348dd5cd0eab9ac5b12a428b74c0c01d) :slightly_smiling_face: